_pages/introduction-developing-skills.md

```diff
@@ -303,52 +301,48 @@ def handle_hello_world_intent(self, message):
         self.speak_dialog("hello.world")
 ```
 
-In the
-
-`handle_hello_world_intent()`
-
- method above, the method receives two _parameters_,
-
- `self`
-
- and
-
- `message`
-
- `self` is the reference to the object itself, and `message` is an incoming message from the `messagebus`. This method then calls the
-
- `speak_dialog()`
-
- method, passing to it the
-
- `hello.world`
-
- dialog. Remember, this is defined in the file "hello.world.dialog".
+In the `handle_hello_world_intent()` method above, the method receives
+two _parameters_: `self` and `message`.  `self` is the reference to the
+object itself, and `message` is an incoming message from the
+`messagebus`.
 
- Can you guess what Mycroft will Speak?
+This method then calls the `speak_dialog()` method, passing to it the
+`hello.world` dialog. Remember, this is defined in the file
+"hello.world.dialog".
 
-You will usually also have a
+Can you guess what Mycroft will Speak?
 
-`stop()`
+##### `stop()` Method
 
-method. This method tells Mycroft what to do if a stop **intent** is detected.
+A **Skill** will usually also have a `stop()` method, which tells
+Mycroft what to do when a stop **Intent** is detected.
 
 ```python
 def stop(self):
     pass
 ```
 
-In the above code block, the [`pass` statement](https://docs.python.org/2/reference/simple_stmts.html#the-pass-statement) is used as a placeholder; it doesn't actually have any function. However, if the **Skill** had any active functionality, the stop() method would terminate the functionality, leaving the *Skill** in a known good state.
+In the above code block, the
+[`pass` statement](https://docs.python.org/2/reference/simple_stmts.html#the-pass-statement)
+is used as a placeholder; it doesn't actually have any function.
+However, if the **Skill** had any active functionality, the `stop()`
+method would terminate that functionality and return the **Skill**
+to a known good state.
 
 ## Simplifying your Skill code with `intent_handler` _decorators_
 
-Your **Skill** code can be simplified using the intent_handler() _decorator_. The major advantage in this approach is that the **Intent** is described together with the method that handles the **Intent**. This makes your code easier to read, easier to write, and errors will be easier to identify.
+Your **Skill** code can be simplified using the `intent_handler` _decorator_.
```

Comment:
The line wrapping on this commit seems to be hard wrapped at 80 characters per line; is that intended?